pennylane/transforms/core/transform_program.py

```diff
@@ -615,3 +615,34 @@ def __call__(
 
         # Reset classical jacobians
         return tuple(tapes), postprocessing_fn
+
+    def __call_jaxpr(
+        self, jaxpr: "jax.core.Jaxpr", consts: Sequence, *args
```

Comment:
This make me really wish we could make `jax` a dependency, and then we could just use `singledispatchmethod`. Oh well.